main.js

```diff
@@ -54,6 +58,26 @@ function startTimer(event) {
   checkInputs(activity);
 };
 
+function startCountDown() {
+  var secondsInput = parseInt(seconds.value);
+  var minutesInput = parseInt(minutes.value);
+  var oneSecond = 1;
+  var oneMinute = oneSecond * 60;
+  var totalTime = (secondsInput * oneSecond) + (minutesInput * oneMinute);
+  var timer = setInterval(function() {
```

Comment:
Timer works but does not pad zeroes when the coundown moves below 10 seconds - need to work out that bug